src/tools/rados/rados.cc

```diff
@@ -2616,6 +2617,15 @@ static int rados_tool_common(const std::map < std::string, std::string > &opts,
       goto out;
     }
 
+    if (rados.get_pool_is_selfmanaged_snaps_mode(src_pool)) {
+      cerr << "WARNING: pool " << src_pool << " has selfmanaged snaps, which are not preserved\n"
+	   << "    by the cppool operation.  This will break any snapshot user.\n"
+	   << "    If you insist on making a broken copy, you can pass\n"
+	   << "    --yes-i-really-mean-it to proceed anyway."
+	   << std::endl;
+      if (!force)
```

Comment:
@liewegas maybe split the cerr message, and have the second part (starting at 'If you insist') dumped only if !force?